src/spatial/math.js

```diff
@@ -728,7 +737,7 @@ Crafty.math.Vector2D = (function () {
      *
      * @public
      * @static
-     * @sign public {Vector2D} tripleProduct(Vector2D, Vector2D, Vector2D, [Vector2D]);
+     * @sign public {Vector2D} tripleProduct(Vector2D a, Vector2D b, Vector2D c, [Vector2D result]);
      * @param {Vector2D} a
      * @param {Vector2D} b
      * @param {Vector2D} c
```

Comment:
Yeah, had trouble thinking of how to describe these.  I believe they're supposed to match the descriptions [here](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setTransform), so I'll copy those.